### DISCUSSION TITLE
feature：private network

### DISCUSSION THREAD

**Discussion by Mydreamandreality** [gravitl/netmaker#2268] [General]
How to use the internal network, I just want to use NetMaker to manage the internal network，Is a private network

Can you give some suggestions, thank you, look forward to your reply

**Comment by cdpb** [comment]
There is a flag in the node settings "is local". With that you could spin up a local only network

**Comment by Mydreamandreality** [comment]
> 节点设置中有一个标志“是本地的”。有了它，您可以启动仅本地网络

Thank you for your reply. My previous description is not accurate. I want to deploy netmaker to the inner network and only manage the internal network. Do you have good suggestions?

**Comment by afeiszli** [comment]
This is currently out of scope. There are ways you could try setting this up on your own as an experiment, but we cannot confirm that it would function.
